test/membrane_http_adaptive_stream/integration_test/sink_bin_integration_test.exs

```diff
@@ -0,0 +1,159 @@
+defmodule Membrane.HTTPAdaptiveStream.SinkBinIntegrationTest do
+  use ExUnit.Case
+
+  import Membrane.Testing.Assertions
+  alias Membrane.{Testing}
+  alias Membrane.Hackney.Source
+
+  @create_references_mode false
+
+  @single_video_track_source [
+    {%Source{
+       location:
+         "https://raw.githubusercontent.com/membraneframework/static/gh-pages/video-samples/test-video.h264",
+       hackney_opts: [follow_redirect: true]
+     }, :H264, "single_video_track"}
+  ]
+  @single_video_track_ref_path "./test/membrane_http_adaptive_stream/integration_test/reference_playlists/single_video_track/"
+  @single_video_track_test_path "./test/membrane_http_adaptive_stream/integration_test/test_playlists/single_video_track/"
+
+  @audio_video_tracks_sources [
+    {%Source{
+       location:
+         "https://raw.githubusercontent.com/membraneframework/static/gh-pages/samples/test-audio.aac",
+       hackney_opts: [follow_redirect: true]
+     }, :AAC, "audio_track"},
+    {%Source{
+       location:
+         "https://raw.githubusercontent.com/membraneframework/static/gh-pages/video-samples/test-video.h264",
+       hackney_opts: [follow_redirect: true]
+     }, :H264, "video_track"}
+  ]
+  @audio_video_tracks_ref_path "./test/membrane_http_adaptive_stream/integration_test/reference_playlists/audio_video_tracks/"
+  @audio_video_tracks_test_path "./test/membrane_http_adaptive_stream/integration_test/test_playlists/audio_video_tracks/"
+
+  defmodule TestPipeline do
+    use Membrane.Pipeline
+    alias Membrane.HTTPAdaptiveStream.{SinkBin, HLS}
+    alias Membrane.HTTPAdaptiveStream.Storages.FileStorage
+    alias Membrane.H264.FFmpeg.Parser, as: H264_Parser
+    alias Membrane.AAC.Parser, as: AAC_Parser
+
+    @impl true
+    def handle_init(options) do
+      sink_bin = %SinkBin{
+        muxer_segment_duration: 2 |> Membrane.Time.seconds(),
+        manifest_module: HLS,
+        target_window_duration: 30 |> Membrane.Time.seconds(),
+        target_segment_duration: 2 |> Membrane.Time.seconds(),
+        persist?: false,
+        storage: %FileStorage{
+          directory: options[:output_dir]
+        }
+      }
+
+      elements =
+        options[:sources]
+        |> Enum.map(fn {source, encoding, track_name} ->
+          [
+            {{:source, track_name}, source},
+            {{:parser, encoding, track_name}, get_parser_for_encoding(encoding)}
+          ]
+        end)
```

Comment:
These are children. Also, you can add SinkBin here if you apply next comment
```suggestion
      children =
        options[:sources]
        |> Enum.flat_map(fn {source, encoding, track_name} ->
          [
            {{:source, track_name}, source},
            {{:parser, encoding, track_name}, get_parser_for_encoding(encoding)}
          ]
        end)
        |> then(&[{:sink_bin, sink_bin} | &1])
```

Reply:
Done